UNRELEASED.md

```diff
@@ -5,3 +5,13 @@
 
 - bpk-component-map:
   - Added the component `BpkMapMarker`.
+
+- bpk-mixins:
+  - Add a mixin for primary color use `bpk-primary-color-property`.
+    ```scss
+    @import '~bpk-mixins';
+
+    .selector {
+      @include bpk-primary-color-property(color);
```

Comment:
Might be worth making it clear that this applies it to `color` too. We could have something like:

```
.selector {
  // Instead of
  // color: $bpk-color-blue-500;
  include bpk-primary-color-property(color);
}
```

Reply:
Sorry, my last comment wasn't clear. What I meant is that I didn't understand at a glance that the `(color)` argument means it applies the mixin to the CSS color property, which is why I recommended an inline comment to clarify.

Reply:
Yeah makes sense, I've changed it as requested. Also happy to change the name of the mixin if you have a better one in mind

Reply:
I can't really think of a better name either.. maybe something like `bpk-apply-primary-color-to(property)` but then it's a bit Objective-C sounding, it wouldn't really fit with the other mixins.

Reply:
What about having two mixins? One for `color` and one for `background-color`?

Reply:
Specifically: `@include bpk-primary-color;` and `@include bpk-primary-background-color;`

Reply:
Yeah not a bad idea Matt. We could always leave this mixin in to cover any edge cases we can't think of (e.g. `border-color`).

Reply:
Scrap that, there are too many variants. Equally `bpk-primary-color-property($property-name)` doesn't fit well with shorthand properties i.e. `border: $width, $style, $color;`

Reply:
I think Shaun's suggestion is pretty good

```sass
@include bpk-apply-primary-color-to(color);

@include bpk-apply-primary-color-to(background);

@include bpk-apply-primary-color-to(fill);

@include bpk-apply-primary-color-to(background-color);

@include bpk-apply-primary-color-to(border-color);
```

Reply:
@k0nserv happy to go with that.